### PR TITLE
feat(DatePicker): add datepicker

### LINKED ISSUE #374: Create Datepicker ui-component
image

### PR DESCRIPTION
**Pull request by zouxuoz**
resolve #374

### CODE REVIEW

File: client/components/ui/DatePicker/DatePicker.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const DatePicker = ({ block, value, onChange }) => (
+  <div {...block()}>
+    <ReactDatePicker fixedHeight inline selected={value} onChange={onChange} />
+  </div>
+);
+
+DatePicker.propTypes = {
+  block: PropTypes.func.isRequired,
+  value: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
```
Comment (radiansz) [inline client/components/ui/DatePicker/DatePicker.js]: Make it instanceOf or, if it wont work, custom validator

Comment (zouxuoz) [inline client/components/ui/DatePicker/DatePicker.js]: fixed

File: packages/sendsay-ui/components/DatePicker/DatePicker.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+.react-datepicker__month {
+  padding: 10px;
+}
+
+.react-datepicker__day-names {
+  padding: 10px 10px 0;
+}
+
+.react-datepicker__day-names,
```
Comment (rosendi) [inline packages/sendsay-ui/components/DatePicker/DatePicker.css]: hmm... flatten it please.

File: packages/sendsay-ui/components/DatePicker/DatePicker.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  display: flex;
+  flex-direction: row;
+}
+
+.react-datepicker__day-name {
+  font-weight: 700;
+  text-transform: capitalize;
+}
+
+.react-datepicker__day-name,
```
Comment (rosendi) [inline packages/sendsay-ui/components/DatePicker/DatePicker.css]: hmm... flatten it please.

File: packages/sendsay-ui/components/DatePicker/DatePicker.css (hunk 2 above)
Comment (rosendi) [inline packages/sendsay-ui/components/DatePicker/DatePicker.css]: I mean I don't want .react-datepicker__day-names, .react-datepicker__week. if they share same css, it should be one element.

File: packages/sendsay-ui/components/DatePicker/DatePicker.css [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+.react-datepicker__day--selected,
+.react-datepicker__day--in-selecting-range,
+.react-datepicker__day--in-range {
+  background-color: var(--primaryColor);
+  border-radius: 2px;
+  color: #fff;
+}
+
+.react-datepicker__day--selected:hover,
```
Comment (rosendi) [inline packages/sendsay-ui/components/DatePicker/DatePicker.css]: :hover and the other pseudos go to `.react-datepicker__day--selected, +.react-datepicker__day--in-selecting-range`

File: packages/sendsay-ui/components/DatePicker/DatePicker.css [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+.react-datepicker__day--in-range:hover {
+  background-color: color(var(--primaryColor) w(25%));
+}
+
+.react-datepicker__day--disabled {
+  color: #ccc;
+  cursor: default;
+}
+
+.react-datepicker__day--disabled:hover {
```
Comment (rosendi) [inline packages/sendsay-ui/components/DatePicker/DatePicker.css]: :hover and the other pseudos go to `.react-datepicker__day--disabled`

**Comment by rosendi** [comment]
1. Put the styles to `src/styles/vendor/DatePicker.css`

2. Remove wrapper component and don't test it since we made contract with ReactDatepicker should be tested. Your test repeats ReactDatepicker's tests.

**Comment by zouxuoz** [comment]
@rosendi fixed

**Comment by rosendi** [comment]
lgtm

**Comment by radiansz** [comment]
lgtm

**Pull request merged**